wgsl/index.bs

```diff
@@ -10774,32 +10780,45 @@ used in address space |C|.
         <th>[=RequiredAlignOf=](|S|, |C|),<br/>
             |C| is not [=address spaces/uniform=]
         <th>[=RequiredAlignOf=](|S|, |C|),<br/>
-            |C| is [=address spaces/uniform=]
+            |C| is [=address spaces/uniform=] and
```

Comment:
This is different from previous language extension changes (e.g. pointer parameters) based on feedback from @jimblandy.